tests/test_ocr.py

```diff
@@ -185,6 +185,38 @@ def test_char_whitelist():
         char_whitelist="0123456789")
 
 
+@requires_tesseract
+@pytest.mark.parametrize("corrections,expected", [
+    # pylint:disable=bad-whitespace
+    # Default ocr output:
+    (None,                                           'OO'),
+    # Corrections string must match entire word:
+    ({'O': '0'},                                     'OO'),
+    ({'OO': '00'},                                   '00'),
+    # Strings are case-sensitive, and they aren't regexes:
+    ({'oo': '00', '[oO]': '0'},                      'OO'),
+    # Regexes do match anywhere:
+    ({re.compile('[oO]'): '0'},                      '00'),
+    # Make sure it tries all the patterns:
+    ({'AA': 'BB', 'OO': '00'},                       '00'),
+    ({re.compile('^O'): '1', re.compile('O$'): '2'}, '12'),
+])
+def test_corrections(corrections, expected):
+    f = load_image('ocr/00.png')
+    print(corrections)
+    assert expected == stbt.ocr(frame=f, mode=stbt.OcrMode.SINGLE_WORD,
```

Comment:
I think you definitely want some tests with multiple words.  Perhaps test `apply_ocr_corrections` directly.  You only need integration tests involving `ocr` (i.e. just to make sure `apply_ocr_corrections` is being called), but it's easy to add more thorough tests by calling `apply_ocr_corrections`.